ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -46,9 +47,9 @@ opts_SetScriptDescription "ReApplyFix Pipeline for MultiRun ICA+FIX"
 #WARNING: this "default" is also used to special case whether filenames have ".<num>k" added to their regstring
 G_DEFAULT_LOW_RES_MESH=32
 
-opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects" '--path'
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all sessions" '--path'
 
-opts_AddMandatory '--subject' 'Subject' 'subject ID' "(e.g. 100610)"
+opts_AddMandatory '--session' 'SessionCross' 'session ID' "(e.g. 100610)" "--subject"
```

Comment:
```suggestion
opts_AddMandatory '--session' 'SessionCross' 'session ID' "(e.g. 100610_V1)" "--subject"
```
I suggest we use an example that makes clearer it is a "session" not a "subject" ID.

Reply:
Ok